io.github.tofudd.lanchat.terminal.metainfo.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
```

Comment:
Just a note – you may want to change it to desktop-application as we currently don't display console-apps on https://flathub.org